.github/workflows/ci.yaml

```diff
@@ -134,7 +134,7 @@ jobs:
 
       - name: Cmd Tests
         run: |
-          ./project/scripts/sbt ";dist/pack; scala3-bootstrapped/compile; scala3-bootstrapped/test ;sbt-test/scripted scala2-compat/* ;stdlib-bootstrapped/test:run ;stdlib-bootstrapped-tasty-tests/test; scala3-compiler-bootstrapped/scala3CompilerCoursierTest:test"
+          ./project/scripts/sbt ";dist/pack; scala3-bootstrapped/compile; scala3-bootstrapped/test ;sbt-test/scripted scala2-compat/* ;stdlib-bootstrapped/test:run ;stdlib-2_13-bootstrapped/test:run ;stdlib-2_13-tasty-bundle/packageBin ;stdlib-bootstrapped-tasty-tests/test; scala3-compiler-bootstrapped/scala3CompilerCoursierTest:test"
```

Comment:
- Could we delete stdlib-bootstrapped? If we have both the community build stdlib and stdlib-2_13-bootstrapped then I don't think it serves a purpose anymore
- Can we merge stdlib-2_13-bootstrapped and stdlib-2_13-tasty-bundle in one project? This might be easier to do when using `mappings` in the build as I propose in my other comment.

Reply:
> * Could we delete stdlib-bootstrapped? If we have both the community build stdlib and stdlib-2_13-bootstrapped then I don't think it serves a purpose anymore

It still serves the purpose of checking that we can compile the stdlib using Scala 3 semantics/signatures (without `-Yscala2-stdlib`). It its also a way to check that we are able to compile all Scala 2 and Scala 3 library sources together. This is not something we need to be able to do right now. It could prevent us from breaking something we might need in the long term.

> * Can we merge stdlib-2_13-bootstrapped and stdlib-2_13-tasty-bundle in one project? This might be easier to do when using mappings in the build as I propose in my other comment.

I removed the bundle project. Now we package only the `.tasty` files in `stdlib-2_13-bootstrapped`.

Reply:
> It still serves the purpose of checking that we can compile the stdlib using Scala 3 semantics/signatures (without -Yscala2-stdlib). It its also a way to check that we are able to compile all Scala 2 and Scala 3 library sources together. 

This is also checked by the stdlib213 community-build project (it relies on https://github.com/scala/scala/blob/2.13.x/project/DottySupport.scala).

Reply:
OK, then I guess we can remove it and only keep one.

Reply:
I rebased and cleaned up the git history of the changes in the build. Now we only modify the definition of `stdlib-bootstrapped`.